### PR TITLE
README.md - config examples

### PR DESCRIPTION
**Pull request by JanneKiiskila**
Some configuration examples, as it's now always that easy to
decipher the Mbed OS advanced configuration system.

More info in: https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/

### CODE REVIEW

**Comment by betzw** [comment]
Thanks for this!

**Comment by betzw** [comment]
Pls. note that it's not a 100% correct. But I will add your suggestions in the next version of the driver.

**Comment by JanneKiiskila** [comment]
Well, that's why it's for review! What's wrong - we'll fix.

**Comment by betzw** [comment]
Just wait for my next version as I would like to add also information regarding FW upgrade.

**Comment by betzw** [comment]
If you are interested at a preview you can refer to https://github.com/betzw/wifi-x-nucleo-idw01m1/blob/betzw_spwfxx_debug_wb/README.md

**Comment by betzw** [comment]
Should be superceeded by latest master version.
@JanneKiiskila, what do you think?

File: README.md [hunk 1]
```diff
@@ -3,3 +3,36 @@
...
+
+``` json
+        "idw0x11": {
+            "expansion.board": "IDW04A1"
+        }
+```
+
+### Adding the HW-workaround for IDW04A1
+
+You can add the HW workaround acivation macro the `macros` section in your `mbed_app.json`.
```
Comment (JanneKiiskila) [inline README.md:34]: Silly typo, acivation -> activation.

Comment (betzw) [reply README.md:34]: Thanks :face_with_head_bandage:

**Comment by betzw** [comment]
Will fix the typo in next release on `master` branch.
What would you think about closing this PR?

**Comment by JanneKiiskila** [comment]
Closing out, changes will come via @betzw 's PR.

**Pull request closed without merging**